Python Tutorial Exercises(Week 3)/#7.py

```diff
@@ -0,0 +1,63 @@
+'''
+Solve Week 2 exercises using while loops instead of for loops
+'''
+#1
+x = [["this", "is", "a", "nested"], ["list", "that"], ["will", "be", "printed", "out"]]
+#                0                         1                         2
+nested_list_one = 0
+nested_list_two = 0
+#len(x)=3 and nested_list_one will never greater than 3 becuse there only 3 list in the x list[0,1,2]. So we use '<' in here.
+while nested_list_one < len(x):
+    #nested_list_two = [ [0,1,2,3], [0,1], [0,1,2,3]  ] . 
+    #len(x[nested_list_one]) = 4444(nested list 1 have 4 items and so on),22,4444.
+    #nested_list_two will never greater than nested_list_one, so use '<' in here.
+    while nested_list_two < len(x[nested_list_one]):
+        print(x[nested_list_one][nested_list_two])
+        nested_list_two += 1
+        #reset loop [1,2,3,4] &[4,4,4,4] if 4 == 4 reset list two and break.
+        if nested_list_two == len(x[nested_list_one]):
```

Comment:
why would you need this break condition if the while loop will handle that already, you should think of a different place where you can reset `nested_list_two = 0`